content/blog/setting-bios-and-storage-controller-properties-with-redfish.md

```diff
@@ -15,88 +16,40 @@ tags:
-
-The important thing to note in this flow is that the final status code and associated message of a property setting is visible **after the system reset**.
-
-## Successful example
-
-In this example, we create a Raid1 storage array of two physical disks using Postman. To achieve this goal, we have to issue a `PUT` to `{{iloURI}}/redfish/v1/Systems/1/SmartStorageConfig/Settings/` with a body (aka payload) similar to the example shown below:
-
-![Body to PUT to .../SmartStorageConfig/Settings/](/img/1-put-body.png "Body to PUT to .../SmartStorageConfig/Settings/")
-
-Upon successful completion of this `PUT` request, the HTTP status return code is `200 OK`, which means that the remote Web server understood what to do with this well-formed payload.
-
-The Body of the HTTP response contains an `error` property with a `SystemResetRequired` message. This property is a Redfish object sent by the Redfish server. It is there to give details about the next step to perform to finish the modification process.
-
-![PUT Return Status](/img/2-put-return-status.png "PUT Return Status")
-
-At this stage of the process, only the staging / "pending zone" of the Smart Array has changed and contains the `PUT` payload. We can verify this assertion by comparing the content of  `{{Subsystem-URI}}/Settings/` with the content of `{{Subsystem-URI/}}`.
-
-In the pending zone (`.../SmartStorageConfig/Settings/`) we can see the payload we sent to the Redfish server. However, in the "running zone" (`.../SmartStorageConfig/`) the `LogicalDrives` array is still empty:
-
-![GET Pending and Current LogicalDrives](/img/3-getpendinglogicadrives.bmp "GET Pending and Current LogicalDrives")
-
-It is now time to reset the server and perform a `GET` of the running zone. In the response body of this operation, the first Redfish object is a `@Redfish.Settings` collection containing a single `MessageID` mentioning `Success`. This single message is synonym of a successful transfer of the "pending zone" in to the "running zone". We will see later in this document what we get in case of an un-successful transfer.
-
-![GET Running zone after server reset](/img/5-getafterreset.png "GET Running zone after server reset")
-
-Further down in this response we find the `LogicalDrives` array containing the Raid1 disk array:
-
-![GET LogicalDrives from running zone](/img/6-getafterreset-2.png "GET LogicalDrives from running zone")
-
-## Unsuccessful example
-
-In order to emphasize the fact that the modification status of properties in the `Bios` and `SmartStorage` subsystems must be done after a system reset, we will study a case where the JSON `PUT` payload is syntactically correct but embedding a value typo (`Raid` instead of `Raid1`) and missing a required key-value (`DataGuard=Disabled`):
```

Comment:
Change the second part of the sentence to:

"... after a system reset, note the case study below where the JSON 'PUT' payload is synttactically correct but embeds a value typo ('Raid' instead of 'Raid1') and is missing a required key-value ('DataGuard=Disabled'):  "